#Define goal's message
string turtle_name
int32 speed
float32 degrees
bool isClockwise
---
#Define result's message
---
#Define feedback's message
turtlesim/Pose pose
